loader/loader_windows.c

```diff
@@ -525,8 +525,8 @@ VkResult windows_get_registry_files(const struct loader_instance *inst, char *lo
                                 foundDuplicate = true;
                             }
                         }
-
-                        if (foundDuplicate == false) {
+                        // Only skip if we are adding a driver and a duplicate was found
+                        if (!is_driver || is_driver && foundDuplicate == false) {
```

Comment:
Need paranthesis for safety?

Reply:
No, but would allow for more clarity of intent, instead of relying on binary operator precedence.